README.md

```diff
@@ -31,7 +31,7 @@ public class App {
     Config config = new Config("http", "localhost:8080");
     WeaviateClient client = new WeaviateClient(config);
     Result<Meta> meta = client.misc().metaGetter().run();
-    if (meta.getError() != null) {
+    if (meta.getError() == null) {
```

Comment:
Even better would be to use here `Result::hasErrors()` method
https://github.com/semi-technologies/weaviate-java-client/blob/main/src/main/java/technology/semi/weaviate/client/base/Result.java#L35-L37

Reply:
pushed something!